packages/igx-templates/igx-ts/projects/_base/files/package.json

```diff
@@ -20,7 +20,6 @@
     "@angular/router": "~15.0.0",
     "hammerjs": "^2.0.8",
     "igniteui-angular": "~15.0.0",
-    "igniteui-theming": "~1.2.0",
     "core-js": "^3.6.5",
     "jszip": "^3.10.0",
```

Comment:
Fairly certain this one is also obsolete, so it can be removed as well :)

Reply:
Also have a strong suspicion for `core-js` is also no longer needed

Reply:
https://github.com/IgniteUI/igniteui-cli/issues/1099 doing it next :)